test/player_dolby_vision_integration.js

```diff
@@ -0,0 +1,187 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Player', () => {
```

Comment:
This should have a specific name. `Player Dolby Vision Integration` or something. It'll make it easier to tell what test suite has the failures.

Reply:
Done!